File: 3st/Juram/programmer/0 떼기.js
function solution(n_str) {
    var answer = '';
    let cnt = 0;
    for(let i=0; i<n_str.length; i++){
        if(n_str[i]==0) cnt++;
        else break;
    }
    answer = n_str.slice(cnt,n_str.length);
    
    return answer;
}

File: 4st/Jisoo/programmer/0 떼기.java
class Solution {
    public String solution(String n_str) {
        int i = 0;
        
        while (i < n_str.length() && n_str.charAt(i) == '0') {
            i++;
        }

        if (i == n_str.length()) {
            return "0";
        }

        return n_str.substring(i);
    }
}
